platform/kube/inject/testdata/auth.cert-dir.yaml.injected

```diff
@@ -49,10 +49,16 @@ spec:
         securityContext:
           runAsUser: 1337
         volumeMounts:
+        - mountPath: /etc/istio
```

Comment:
Isn't "etc" for configuration files? Why would we not put configuration into "etc"?